bin/test_travis.sh

```diff
@@ -119,9 +119,15 @@ fi
 if [[ "${TEST_SYMPY}" == "true" ]]; then
     cat << EOF | python
 print('Testing SYMPY, split ${SPLIT}')
-import sympy
-if not sympy.test(split='${SPLIT}'):
+import sympy, coverage
+cov = coverage.Coverage()
+cov.start()
+if not sympy.test(split='${SPLIT}', subprocess=False):
```

Comment:
With `subprocess=False` you won't get the `PYTHONHASHSEED` environment variable at the start of the tests that reproduces the hash randomization.  Hash randomization itself is always enabled, running the tests in a subprocess just lets us set the seed so that it is reproducible, which is very useful for rare test failures caused by specific hash values.